modules/seqtk/rename/main.nf

```diff
@@ -0,0 +1,41 @@
+process SEQTK_RENAME {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::seqtk=1.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/seqtk:1.3--h5bf99c6_3' :
+        'quay.io/biocontainers/seqtk:1.3--h5bf99c6_3' }"
+
+    input:
+    tuple val(meta), path(sequences)
+    val(name)
```

Comment:
As far as I see this is used only for the prefix. This can be set in a pipeline in the `modules.config` with `ext.prefix`. I would recommend removing it here